lib/utils/tests/age.test.ts

```diff
@@ -0,0 +1,25 @@
+import { isUnderEighteen } from "../age";
+
+beforeAll(() => {
+  jest.useFakeTimers("modern");
+  jest.setSystemTime(new Date("2021-12-25"));
```

Comment:
Do we need to reset the system time for other tests?

Reply:
useRealTimers resets the date